### PR TITLE
Add detail popup page

### PR DESCRIPTION
**Pull request by dennis-every**
For this PR we completed the following:
- Add JS array data for works
- Remove static works html, add js to generate works cards on window.load
- Add JS for card: worksActivateLinkHandler, cardClickHandler
- Add work modal html, css, and JS to open and close modal
- Update cardClickHandler function to show card in modal
- Add remove modal card from dom after click on close modal
- Update modal work card to align with mobile modal design
- Update modal work card to align with desktop modal design
- Add collaborator to readme

### CODE REVIEW

File: javascript/app.js [hunk 1]
```diff
@@ -22,3 +23,194 @@ function closeMenuHandler() {
...
+    description:
+      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+    technologies: ['html', 'css', 'javascript', 'github', 'ruby', 'Bootstraps'],
+    urlMobile: '/images/work_1.jpg',
+    urlDesktop: '/images/work_1_lg.png',
+    urlModal: '/images/work_modal.png',
+    alt: 'work_tonic_daily_personalized_reads',
+    live: 'https://dennis-every.github.io/',
+    source: 'https://github.com/dennis-every/portfolio',
+  },
```
Comment (BenMukebo) [inline javascript/app.js:41]: - 👏🏻 👏🏻 Excellent work implements an array of objects to store JavaScript objects containing the information for each project. 😉  💯

File: javascript/app.js [hunk 2]
```diff
@@ -22,3 +23,194 @@ function closeMenuHandler() {
...
+                <li>${work.technologies[0]}</li>
+                <li>${work.technologies[1]}</li>
+                <li>${work.technologies[2]}</li>
+              </ul>
+            </div>
+            <div class='btn-container'>
+              <button type='button' class='btn'>
+                <a href=${work.live}>See Project</a>
+              </button>
+            </div>
```
Comment (BenMukebo) [inline javascript/app.js]: - [ ] Great job so far 👏🏻 However all the `See Project` doesn't work as shown on the video screen 👇🏻 Actually they are working as an internal link to the home section as shown on the screenshot video. When I click on one, of them the URL link change to this 👉🏻   `https://dennis-every.github.io/#`/. Kindly let's fix this and make sure that: 
-  When the user clicks the button to check project details, the popup with details about the project appears.✔️   
- When the user clicks the close button, the popup disappears ✔️ 



[Screencast from 12-21-2022 05:37:39 AM.webm](https://user-images.githubusercontent.com/79058364/208816682-79f0ce76-678c-401b-8c6d-7e7e9a8665f5.webm)


- [ ] Please make sure The popup follows the design for both desktop and mobile versions from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colours, images, text, margins).

**Review by BenMukebo — CHANGES_REQUESTED** [review]
### Hi @Team,

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

### To highlight 💪🏻

- No linter errors. ✔️ 
- GitHub flow was correctly used.  ✔️   
- The [JavaScript](https://github.com/microverseinc/linters-config/tree/master/javascript) and [HTML&CSS](https://github.com/microverseinc/linters-config/tree/master/html-css) linters is set correctly  ✔️ 
 👍  Good job!


## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_


_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [OPTIONAL]  Please let's make sure that contributions to a group project are distributed equally among the members (commit messages should be fairly equal).

Cheers and Happy coding!👏👏👏 💻  
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @BenMukebo when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by levy002 — APPROVED** [review]
## Hello @dennis-every 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Status: APPROVED :fire:  :fire: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**